rpcserver.go

```diff
@@ -322,9 +322,11 @@ func newRPCServer(listenAddrs []string, maxPost, maxWebsockets int64) (*rpcServe
 	// Setup TLS if not disabled.
 	listenFunc := net.Listen
 	if !cfg.DisableServerTLS {
-		// Check for existence of cert file and key file
-		if !fileExists(cfg.RPCKey) && !fileExists(cfg.RPCCert) {
-			// if both files do not exist, we generate them.
+		// Check for existence of cert file and key file.  Generate a
+		// new keypair if both are missing.
+		_, e1 := os.Stat(cfg.RPCKey)
```

Comment:
It gets needlessly messy because of the extra error paths for io errors not due to a missing file.

```
keyExists, err1 := cfgutil.FileExists(cfg.RPCKey)
certExists, err2 := cfgutil.FileExists(cfg.RPCCert)
if err1 == nil && err2 == nil && !keyExists && !certExists {
    // ...
}
`